modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ScalaPlayFrameworkServerCodegen.java

```diff
@@ -0,0 +1,264 @@
+package org.openapitools.codegen.languages;
+
+import io.swagger.v3.oas.models.media.ArraySchema;
+import io.swagger.v3.oas.models.media.Schema;
+import org.openapitools.codegen.*;
+import org.openapitools.codegen.utils.ModelUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.openapitools.codegen.utils.StringUtils.camelize;
+
+public class ScalaPlayFrameworkServerCodegen extends AbstractScalaCodegen implements CodegenConfig {
+    public static final String TITLE = "title";
+    public static final String IMPL_STUBS = "implStubs";
```

Comment:
I think this name may be confusing, as it indicates generation of stubbed implementations... but this is a default behavior in most generators that support interfaces.

I'm wondering if this is the same option in the Java Play generator, `controllerOnly`:

> Whether to generate only API interface stubs without the server files.

If so, we may want to align the option name to avoid confusion for those evaluating the Play generation with different languages. This wouldn't be necessary in this PR, though. I just wanted to document the thought.

Reply:
`implStubs` is similar to `controllerOnly` in that they both affect whether stubs are generated, but `controllerOnly` also affects the interface in java-play-framework, while in this generator I wish to allow generating the trait regardless of the stubs. Perhaps I should make both configurable?

Also in the spirit of keeping things similar, I'll rename `apiFutures` to `supportAsync`.